.azure-pipelines/generation-templates/php-kiota.yml

```diff
@@ -9,3 +9,12 @@ steps:
     BuildConfiguration: $(buildConfiguration)
     OutputFullPath: $(kiotaDirectory)/output/*
     RepoModelsDir: $(Build.SourcesDirectory)/${{ parameters.repoName }}/src/Generated/
+
+# right now the script runs fine with PHP 8.1 which is what is installed by default on the agent
+# https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md
+# but we should use a setup task to install the right version of PHP
+# however, there does not seem to exist a setup-php task (equivalent to UseDotnet)
+
+- pwsh: 'php $(Build.SourcesDirectory)/${{ parameters.repoName }}/scripts/BumpStableSdkVersion.ps1'
```

Comment:
```suggestion
- pwsh: 'php $(Build.SourcesDirectory)/${{ parameters.repoName }}/scripts/BumpStableSdkVersion.php'
```

Reply:
Would it be worthwhile to log an issue to streamline all scripts to PowerShell?

Reply:
I think the structure of the different project files are different across languages ranging from xml/json formats, if a given language is having scenario that is easiest solved in a certain way that isn't powershell, I think it makes sense to go that way.